Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+        response = client.mute_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.MuteIncident",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def unmute_incident_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+        response = client.unmute_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.UnmuteIncident",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def close_incident_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+        response = client.close_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.CloseIncident",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def consult_incidents_updates_through_rest_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    offset = args.get("offset")
+    items = args.get("items")
+    time = args.get("time")
+
+    response = client.consult_incidents_updates_through_rest_request(
+        offset, items, time
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.ConsultIncidentsUpdatesThroughRest",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def fetch_incidents(client: Client, first_fetch_time, last_run):
+    items = 30
+    time = 4
```

Comment:
for what you use those variables? can we get them from the integration parameters?

Reply:
the items and time variables were promoted as a parameters, solved in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7